packages/client/src/types.ts

```diff
@@ -11,6 +11,10 @@ export interface Dictionary<T> {
   [key: string]: T;
 }
 
+export enum ArchiveFormat {
+  Tgz = 'tgz',
+}
```

Comment:
I'm not sure we want to use an enum because this is one of the quirky TS features that works differently depending on using TS proper or some other compiler. And it also behaves differently if adding a `const enum`.

How about separating the valid values (runtime) from the type (build time) like this:

https://github.com/vercel/vercel/blob/181a492d912b4c11970f824afc43638519a5ef7f/packages/routing-utils/src/index.ts#L26-L35